test/gtest/ucp/test_ucp_am.cc

```diff
@@ -50,6 +50,10 @@ class test_ucp_am_base : public ucp_test {
 
     virtual void init()
     {
+        if (has_transport("ud_v") && ucs::is_aws()) {
+            UCS_TEST_SKIP_R("AWS EFA/UD: low SGE count per WQE AM IOV sends");
```

Comment:
does it mean UCP AM is not really supported with efa/ud?

Reply:
yes I think that's valid concern, my understanding is that unless we somehow linearize the send operation for the EFA case, we will not be able to support all cases.

Reply:
AFAIR many SGEs are only needed for AM short. Can we make sure that EFA iface does not set AM SHORT capability?